pkg/api/v1/helpers.go

```diff
@@ -277,6 +277,10 @@ const (
 	// an object (e.g. secret, config map) before fetching it again from apiserver.
 	// This annotation can be attached to node.
 	ObjectTTLAnnotationKey string = "node.alpha.kubernetes.io/ttl"
+
+	// AffinityAnnotationKey represents the key of affinity data (json serialized)
+	// in the Annotations of a Pod.  TODO remove in 1.7
```

Comment:
We might not remove this in 1.7. I would re-phrase to "TODO remove when alpha support for affinity is removed" (here and everywhere else in the PR where you reference 1.7).